datasets/caldor-fire-characteristics-burn-severity.data.mdx

```diff
@@ -0,0 +1,69 @@
+---
+id: caldor-fire-behavior-burn-severity
+name: "Caldor Fire Behavior and Burn Severity"
+description: "Assets describing the progression and active fire behavior of the 2021 Caldor Fire in California."
+media:
+  src: ::file ./caldor-fire--dataset-cover.jpg
+  alt: Hillside engulfed by a wildfire.
+  author:
+    name: Marek Piwnicki
+    url: https://unsplash.com/photos/WiZOyYqzUss
+thematics:
+  - eis
+layers:
+  - id: caldor-fire-behavior
+    stacCol: caldor-fire-behavior
+    name: Fire Behavior
+    type: raster
+    description: ""
+    initialDatetime: newest
+    zoomExtent:
+      - 10
+      - 20
+    sourceParams:
+      colormap_name: inferno_r
+      rescale:
+        - -41
+        - 98
+    legend:
+      type: gradient
+      min: "-41"
+      max: "98"
+      stops:
+        - "#FBB41A"
+        - "#BB3754"
+        - "#781D6D"
+        - "#34095F"
+  - id: caldor-fire-burn-severity
+    stacCol: caldor-fire-burn-severity
+    name: Burn Severity
+    type: raster
+    description: "Soil burn severity (SBS) conditions."
+    initialDatetime: newest
+    zoomExtent:
+      - 10
+      - 20
+    sourceParams:
+      colormap_name: inferno_r
+      rescale:
+        - 0
+        - 5
+    legend:
+      type: gradient
+      min: "0"
+      max: "5"
+      stops:
+        - "#FBB41A"
+        - "#BB3754"
+        - "#781D6D"
+        - "#34095F"
+  
+---
+
+<Block>
+<Prose>
+  ## About
+
+  This dataset describes the progression and active fire behavior of the 2021 Caldor Fire in California, analyzed via the algorithm detailed in https://www.nature.com/articles/s41597-022-01343-0. It includes an extra layer detailing the soil burn severity (SBS) conditions provided by the [Burned Area Emergency Response](https://burnseverity.cr.usgs.gov/baer/) team.
```

Comment:
@eorland Take a look at this text.

Reply:
Text looks good! I would _maybe_ change the words "analyzed via" to the phrase "as recorded by" but this really is me splitting hairs.